test/parallel/test-runner-coverage.js

```diff
@@ -428,3 +428,72 @@ test('coverage with included and excluded files', skipIfNoInspector, () => {
   assert.strictEqual(result.status, 0);
   assert(!findCoverageFileForPid(result.pid));
 });
+
+test('coverage with a minimum line threshold', skipIfNoInspector, () => {
```

Comment:
I think we also need a test showing `--experimental-test-coverage` working with a value.